docs/development/connectors.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-A connector in OpenCTI is a service that runs next to the platform and can be implemented in almost any programming language that has STIX2 support. Connectors are used to extend the functionality of OpenCTI and allow operators to shift some of the processing workload to external services. To use the conveniently provided [OpenCTI connector SDK](https://github.com/OpenCTI-Platform/client-python) you need to use **Python3** at the moment.
+A connector in OpenCTI is a service that runs next to the platform and can be implemented in almost any programming language that has STIX2 support. Connectors are used to extend the functionality of OpenCTI and allow operators to shift some of the processing workload to external services. To use the conveniently provided [OpenCTI connector SDK](https://github.com/OpenCTI-Platform/opencti/tree/master/client-python) you need to use **Python3** at the moment.
```

Comment:
STIX2 should be replaced by STIX 2.0 or STIX 2.1 (need to check the version)

Reply:
```suggestion
A connector in OpenCTI is a service that runs next to the platform and can be implemented in almost any programming language that has STIX 2.1 support. Connectors are used to extend the functionality of OpenCTI and allow operators to shift some of the processing workload to external services. To use the conveniently provided [OpenCTI connector SDK](https://github.com/OpenCTI-Platform/opencti/tree/master/client-python) you need to use **Python3** at the moment.
```